### PR TITLE
Added Instructions

### PR DESCRIPTION
**Pull request by rishabh-vasudevan**
I have added the specifications and instructions that the user can follow to submit their voice samples

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,71 @@
...
 
 This is a dataset of voice samples for our wake word detection of "Hey Rodney".
+
+## Steps to Record audio data and upload it on github
+
+## Specifications:
+
+| __Phrase__ | "Hey Rhodney"   |
+| ------------- | ------------- |
+| __Number of Samples__  | 3  |
```
Comment (ctron) [inline README.md]: Maybe it would be good to clarify here if you want to have three files, or three samples in a single file.

File: README.md [hunk 2]
```diff
@@ -1,3 +1,71 @@
...
+| __Sample Rate__ | 16 KHz  |
+| __Number of Channels__ | 1 |
+| __Encoding__ | Signed 16-bit PCM |
+
+
+
+## Step 1: Record the Audio
+
+### Steps to implement in audacity:
+1. Open audacity
```
Comment (ctron) [inline README.md]: Where do I get "Audacity" from?

File: README.md [hunk 3]
```diff
@@ -1,3 +1,71 @@
...
+### Steps to implement in audacity:
+1. Open audacity 
+2. Set Project rate to 16000 ( In the bottom left ) 
+3. Set the recording channel 1 (mono) recording channel ( which is in the bar just above the timestamps )
+4. Record your voice saying *“Hey Rodney”*
+
+
+__Export the audio file in the form of a .wav file__:<br> 
+*Create a dedicated folder to store all recordings*
+1. After completing the recording
```
Comment (ctron) [inline README.md]:
```suggestion
1. After completing each recording 
```

File: README.md [hunk 4]
```diff
@@ -1,3 +1,71 @@
...
+3. Set the recording channel 1 (mono) recording channel ( which is in the bar just above the timestamps )
+4. Record your voice saying *“Hey Rodney”*
+
+
+__Export the audio file in the form of a .wav file__:<br> 
+*Create a dedicated folder to store all recordings*
+1. After completing the recording 
+2. Go to File
+3. Export
+4. Export as *.wav*
```
Comment (ctron) [inline README.md:40]: Maybe note the "encoding" drop down? Which encoding should be selected.

File: README.md [hunk 5]
```diff
@@ -1,3 +1,71 @@
...
+
+__Windows__: 
+- Right click on the folder
+- Send to  
+- Compressed (zipped) 
+- Folder
+
+__Linux__: 
+
+```zip -r [Name of the new folder.zip] [Name of the folder that contains the voice samples]```
```
Comment (ctron) [inline README.md]: Something seems off with the formatting from here on. I think you need to put the ``` in different lines.

File: README.md [hunk 6]
```diff
@@ -1,3 +1,71 @@
...
+- Upload the files 
+- Click “Zip Files” 
+- Click “save zip file”
+
+
+## Step 3: Upload the data on GitHub
+
+### Upload the folder to the GitHub repository *“hey-rodney-dataset”* by creating an issue:
+
+- Open the url : https://github.com/rishabh-vasudevan/hey-rodney-dataset
```
Comment (ctron) [inline README.md]: Don't we use `https://github.com/drogue-iot/hey-rodney-dataset`?

File: README.md [hunk 7]
```diff
@@ -1,3 +1,71 @@
...
+- Go to the issues tab on the top
```
Comment (ctron) [inline README.md]: I think you could directly link to the issues page. Maybe even to the "New Issue" page.

File: README.md [hunk 8]
```diff
@@ -1,3 +1,71 @@
...
+
+
+## Step 3: Upload the data on GitHub
+
+### Upload the folder to the GitHub repository *“hey-rodney-dataset”* by creating an issue:
+
+- Open the url : https://github.com/rishabh-vasudevan/hey-rodney-dataset
+- Go to the issues tab on the top  
+- Click on New Issue 
+- Make the title your name ( if you are comfortable sharing it ) or just say audio sample
```
Comment (ctron) [inline README.md]: I don't think a real name is required.

File: README.md [hunk 9]
```diff
@@ -1,3 +1,71 @@
...
+## Step 3: Upload the data on GitHub
+
+### Upload the folder to the GitHub repository *“hey-rodney-dataset”* by creating an issue:
+
+- Open the url : https://github.com/rishabh-vasudevan/hey-rodney-dataset
+- Go to the issues tab on the top  
+- Click on New Issue 
+- Make the title your name ( if you are comfortable sharing it ) or just say audio sample  
+- Drag and drop the file to the place where it says to leave a comment  
+- You can also add your age, gender, nationality ( if you are comfortable with sharing it )
```
Comment (ctron) [inline README.md]: I think that is very personal information. In any case, there should be some kind of instruction on what and how (which form) to provide it.

**Review by ctron — CHANGES_REQUESTED** [review]
Thanks for the PR!

We definitely need to mention that the audio samples will be licensed under the creative commons (CC0 aka Public Domain).

Regarding creating an issue, maybe a GitHub issue template would be helpful here.

File: README.md (hunk 2 above)
Comment (rishabh-vasudevan) [reply README.md]: I will add the steps to download audacity

File: README.md (hunk 6 above)
Comment (rishabh-vasudevan) [reply README.md]: copied the wrong link by mistake

File: README.md (hunk 7 above)
Comment (rishabh-vasudevan) [reply README.md]: yes that actually will be better

File: README.md (hunk 9 above)
Comment (rishabh-vasudevan) [reply README.md]: I have added this only for the statistics. We can remove this part

File: README.md [hunk 10]
```diff
@@ -1,3 +1,82 @@
 # Hey Rodney dataset
 
 This is a dataset of voice samples for our wake word detection of "Hey Rodney".
+
+All the audio samples will be licensed under the __Creative Commons Zero v1.0 Universal__ (CC0 aka Public Domain)
```
Comment (ctron) [inline README.md]:
```suggestion
All the audio samples will be licensed under the __Creative Commons Zero v1.0 Universal__ (CC0 aka Public Domain). Only upload samples containing your own voice!
```

**Review by ctron — APPROVED** [review]

**Pull request merged**